settings/base.py

```diff
@@ -143,7 +143,7 @@
 
 BASE_DIR = os.path.abspath(os.path.dirname(os.path.dirname(__file__)))
 
-LOCALE_PATHS = [os.path.join(BASE_DIR, "openassessment", "locale")]
+LOCALE_PATHS = [os.path.join(BASE_DIR, "openassessment", "conf", "locale")]
```

Comment:
If I'm not mistaken, this is the correct configuration:

```suggestion
LOCALE_PATHS = [os.path.join(BASE_DIR, "openassessment", "conf/locale")]
```

Reply:
The result is the same for both @OmarIthawi . I guess the error will disappear if we run the test again, it's codecov issue

Reply:
Oh, my bad. It's a `join` operation. Looks great.